editor/doc/doc_dump.cpp

```diff
@@ -111,7 +111,7 @@ void DocDump::dump(const String &p_file) {
 
 		for (List<MethodInfo>::Element *E = method_list.front(); E; E = E->next()) {
 			if (E->get().name == "" || E->get().name[0] == '_')
-				continue; //hiden
+				continue; //hidden
```

Comment:
We should consider requiring a consistent format for these. Instead of `//hidden` it should be `// Hidden`

Reply:
@aaronfranke I made an issue that talked about that and other code inconsistencies some time ago, pretty much went nowhere: https://github.com/godotengine/godot/issues/16521